client/src/pages/SinglePost.js

```diff
@@ -0,0 +1,188 @@
+import React, { useContext, useState, useRef } from 'react';
+import gql from 'graphql-tag';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+import moment from 'moment';
+import {
+  Button,
+  Card,
+  Form,
+  Grid,
+  Image,
+  Icon,
+  Label
+} from 'semantic-ui-react';
+
+import { AuthContext } from '../context/auth';
+import LikeButton from '../components/LikeButton';
+import DeleteButton from '../components/DeleteButton';
+import MyPopup from '../util/MyPopup';
+
+function SinglePost(props) {
+  const postId = props.match.params.postId;
+  const { user } = useContext(AuthContext);
+  const commentInputRef = useRef(null);
+
+  const [comment, setComment] = useState('');
+
+  const {
+    data: { getPost }
+  } = useQuery(FETCH_POST_QUERY, {
```

Comment:
const {
    data: { getPost }
   ={ } } = useQuery(FETCH_POST_QUERY, {
    variables: {
      postId
    }
  });

Reply:
one curly braces at line no. 29 is not correct  its not working